### ISSUE TITLE
Basler boost boA5328-100cm device adaptors

### ISSUE THREAD

**Issue by jamesliu1990** [czbiohub/coPylot#132]
## FEATURE:

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by AhmetCanSolak** [comment]
We can start think about moving this forward @jamesliu1990 , feel free to invite other related individuals to this issue if there is any need for discussion before starting the implementation of the basler device adaptor. If not feel free to start your implementation and open your PR, we can discuss on the implementation and get it merge within 0.0.1 milestone.

**Comment by talonchandler** [comment]
FWIW [`pyCameras`](https://github.com/czbiohub/pycameras) has some support for Basler cameras. [`py_basler.py`](https://github.com/czbiohub/pyCameras/blob/master/py_cameras/py_basler.py) might provide a good starting point. 

@emilyhuynh98 @Axel-Jacobsen do you know what basler models this has been developed against?

**Comment by emilyhuynh98** [comment]
yep, we developed `py_basler.py` for Basler dart daA1600-60um, but it is just a convenient wrapper function for pypylon so it should be generalizable to other models
